content/developer/reference/frontend/javascript_modules.rst

```diff
@@ -73,8 +83,9 @@ For example, let us consider the following module, located in :file:`web/static/
   }
 
 Note the comment in the first line: it describes that this file should be converted.
-Any file without this comment will be kept as-is (which will most likely be an
-error). This file will then be translated into an Odoo module that look like this:
+This comment is optional for files under `static/src` and `static/tests`.
+Elsewhere, any file without this comment will be kept as-is. This file will then be
```

Comment:
this part where we describe how it is transpiled should be put on top, even before explaining how to disable it.

Here's how I see it:
 - intro (we use native modules, simpler, IDE...)
 - but we transpile everything in src and tests in odoo modules -> exemple de code transpilé
 - can be disabled with odoo-module ignore (odoo will check first line blablabla)
 - in other folders, no transpiler by default, can be enable with odoo-module